### DISCUSSION TITLE
LOVE this system! STL or SVG for WC panels, themselves?

### DISCUSSION THREAD

**Discussion by mamedarling** [aderusha/DDD-Printable-Wall-Control-System#89] [Q&A]
Hi there! First, thank you for all the amazing work that went into creating these pieces. I'm so excited to have found them! :)

I've got 16 of the powder-coated steel panels at home. All are white. I'd like to be able to change colors for at least a few of them and imagine using a vinyl or pla skin would be better than trying to paint them. 

Does anyone happen to have an SVG or SLT file for the regular panels that I could use? If not, I'll get to work on one and be sure to share it. Since I've got no CAD/modeling experience, was just hoping one might live somewhere - but maybe this is a good reason to jump in and start learning ;)

**Comment by aderusha** [comment]
Added to the repo just for you :D

https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/Wall%20Control%20Panel%20Model.stl
